modules/specifying-oc-log-levels.adoc

```diff
@@ -8,22 +8,37 @@
 
 You can investigate OpenShift CLI (`oc`) issues by increasing the command's log level.
 
+The {product-title} user's current session token is typically included in logged `curl` requests where required. You can also obtain the current user's session token manually, for use when testing aspects of an `oc` command's underlying process step by step:
```

Comment:
These sentences are moved from [within the procedure](https://github.com/openshift/openshift-docs/pull/63430/files#diff-a7b187ebf9b024b5394cbe234a9fc72f1ec29b101c1fd38b1b7f9acd2ef8b960L24) to here.